File: fix_layouts.py
import os
import re

pages_dir = os.path.join(os.path.dirname(__file__), 'src', 'pages')

for filename in os.listdir(pages_dir):
    if filename.endswith('.jsx'):
        filepath = os.path.join(pages_dir, filename)
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
            
        if 'import Layout from' in content or '<Layout>' in content:
            print(f"Fixing {filename}...")
            # Remove import
            content = re.sub(r"import Layout from ['\"]\.\./components/Layout['\"];?\r?\n?", "", content)
            
            # Replace <Layout> tags
            content = content.replace('<Layout>', '<>')
            content = content.replace('</Layout>', '</>')
            
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)

print("Done!")

File: fix_layouts.js
const fs = require('fs');
const path = require('path');

const dir = path.join(__dirname, 'src', 'pages');
const files = fs.readdirSync(dir).filter(f => f.endsWith('.jsx'));

files.forEach(file => {
  const filePath = path.join(dir, file);
  let content = fs.readFileSync(filePath, 'utf8');
  
  if (content.includes('import Layout from') || content.includes('<Layout>')) {
    console.log(`Fixing ${file}...`);
    // Remove the import line
    content = content.replace(/import Layout from ['"]\.\.\/components\/Layout['"];?\r?\n?/g, '');
    
    // Replace <Layout> with <>
    content = content.replace(/<Layout>/g, '<>');
    
    // Replace </Layout> with </>
    content = content.replace(/<\/Layout>/g, '</>');
    
    fs.writeFileSync(filePath, content, 'utf8');
  }
});
console.log('Done!');
